frontend/src/router/index.js

```diff
@@ -92,6 +99,20 @@ const routes = [
   },
 ]
 
+function hasPermission(permission) {
+  return store.getters['auth/hasPermission'](permission)
+}
+
+function handleAuth(to, next, permission) {
+  const permissions = store.getters['auth/getPermissions']
```

Comment:
I think this is unused, and the `hasPermission(permission) function is just single line.

Could we remove the function and replace this line with something like
```
const hasPermission =  store.getters['auth/hasPermission'](permission)
if (hasPermission) { ... }
```

Reply:
Unused code was a leftover from some earlier debugging. The function definition is definitely not necessary...I'd just copied it from other components.